nbs/text.util.ipynb

```diff
@@ -329,7 +338,13 @@
     "    return symbols_to_sequence(cleaned_text, symbol_set=symbol_set, ignore_symbols=[])\n",
     "\n",
     "\n",
-    "def text_to_sequence(text, cleaner_names, p_arpabet=0.0, symbol_set=DEFAULT_SYMBOLS):\n",
+    "def text_to_sequence(\n",
```

Comment:
should we synthesize text_to_sequence with prepare_input_sequence from uberduck_py?

Reply:
Yep I'm down, good idea.